src/dotless.Core/Parser/Tree/Ruleset.cs

```diff
@@ -279,6 +281,50 @@ protected void EvaluateRules(Env env)
             env.Frames.Pop();
         }
 
+        /// <summary>
+        /// Merges properties with merge operators '+' and '+_'
+        /// </summary>
+        /// <returns>New merged ruleset</returns>
+        private NodeList MergeProperties()
+        {
+            var groups = new Dictionary<string, List<Rule>>();
+            var sorted = new List<Node>();
+            foreach (var node in Rules)
+            {
+                var rule = node as Rule;
+                if (rule == null || rule.Merge == "")
+                {
+                    sorted.Add(node);
+                    continue;
+                }
+                var key = $"{rule.Name},{rule.Important},{rule.Merge}";
+                if (!groups.ContainsKey(key))
+                {
+                    groups[key] = new List<Rule> {rule};
+                    sorted.Add(node);
+                }
+                else
+                {
+                    if (rule.Merge == "")
```

Comment:
Is this a possible branch? Looks like the check on line 295 already covers this case.